README.md

```diff
@@ -55,8 +55,8 @@ that are available in the spreadsheet.
 
 ## Importation job
 
-This bundle allows you to import the [init.xslx](./Resources/fixtures/minimal/init.xlsx) file directly
-in the UI through Import > Import jobs.
+This bundle allows you to import products file directly in the UI through Import > Import jobs.
+Note the init.xlsx import is available in this UI. It should not be used as import (only as initialization file).
```

Comment:
Please note that the **init.xlsx import** is also available via the UI. However, it should not be used as an import system for entities available within this file (families, categories, etc.) once the catalog structure has been set.